tools/serverpod_cli/lib/src/commands/generate.dart

```diff
@@ -66,9 +66,11 @@ class GenerateCommand extends ServerpodCommand {
         config: config,
         endpointsAnalyzer: endpointsAnalyzer,
       );
-    } else {
-      log.info('Done.',
-          style: const TextLogStyle(type: AbstractStyleType.success));
+    } else if (!hasErrors) {
+      log.info('Project generated!',
```

Comment:
I think perhaps the "Done." text is better. It's not the project that is generated, it's code in the project that has been generated.